cmake/pkgconfig.in

```diff
@@ -6,6 +6,6 @@ Name: libOSRM
 Description: Project OSRM library
 Version: v@OSRM_VERSION_MAJOR@.@OSRM_VERSION_MINOR@.@OSRM_VERSION_PATCH@
 Requires:
-Libs: -L${libdir} -losrm
+Libs: -L${libdir} -losrm @CMAKE_EXE_LINKER_FLAGS@
```

Comment:
We need to report `-Wl,-rpath` to fix the build problem. All three variables are the same so there is no difference: https://github.com/Project-OSRM/osrm-backend/blob/master/CMakeLists.txt#L386-L388